libraries/src/Form/FormFactory.php

```diff
@@ -42,4 +42,4 @@ public function createForm(string $name, array $options = []): Form
 
         return $form;
     }
-}
+}
```

Comment:
```suggestion
}

```
Revert end of file change.

Reply:
This and the other one will also fix PHPCS failing.